bokehjs/src/coffee/common/plot.coffee

```diff
@@ -62,6 +74,11 @@ class PlotView extends ContinuumView
 
     @canvas_view.render()
 
+    # If requested, try enabling webgl
+    if @mget('webgl') or window.location.search.indexOf('webgl=1') > 0
```

Comment:
The motivation for overriding `mget` via this mechanism is to allow users to easily turn on/off webgl support. Like, hey, panning this is a bit slow, let's turn webgl on. The plot author cannot always anticipate whether webgl is best turned on or not (and for the time being you probably don't want to turn webgl on by default). Also for debugging it is very convenient to quickly switch to see the differences. 

Using zero, I guess because I was inspired by github's `?w=0` and just took it over. We could change that, although I don't think it matters much; JavaScript is not strict on types anyway :)

As for generalizing this idea for other plot props: I am not sure that this is a good idea, because it breaks down if there are multiple plots on one page. With the webgl flag you simply enable webgl for all plots on the page, which is usually ok.